lib/resolver/live-resolver-query.js

```diff
@@ -1,4 +1,9 @@
 export default function ({ type, target }) {
+  if (type == 'npm') {
+    // If the target looks like 'foo/bar.js', pretend it is 'foo' instead. See
+    // https://github.com/OctoLinker/browser-extension/issues/93
+    target = target.split('/')[0];
+  }
```

Comment:
Oh, that makes sense. I've moved the logic into `javascript-universal.js`. Does that seem like an appropriate place for it?